src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -9634,3 +9634,115 @@ void ObjectMgr::LoadSceneTemplates()
 
     TC_LOG_INFO("server.loading", ">> Loaded %u scene templates in %u ms.", count, GetMSTimeDiffToNow(oldMSTime));
 }
+
+void ObjectMgr::LoadConversationTemplates()
```

Comment:
All queries here are used only in this method and only during startup, there is no need to make them prepared statements (reloading is seen as development purpose, should not affect deciding whether to use a prepared statement or not)